process/npc/LlamaBanker.lua

```diff
@@ -374,7 +374,7 @@ Handlers.add(
     print('Account: ' .. account .. ', Balance: ' .. balance)
 
     local message = 'Psst, you currently have ' .. FormatLlamaTokenAmount(balance) .. ' $LLAMA coins!'
-    message = message .. ' Go to ao-bazar.g8way.io to view and trade your ao tokens.'
+    message = message .. ' Go to ao-bazar.g8way.io or www.permaswap.network/#/ao/LLAMA-AR to view and trade your ao tokens.'
```

Comment:
How about just “permaswap.network”? Main page should be enough I think

Reply:
ok, updated